### ISSUE TITLE
Checking bytecode of existing file

### ISSUE THREAD

**Issue by b5y** [myint/pydiff#2]
Hello,

It is very good idea to compare two files by their bytecode. But, sometimes bytecode of file is not doing the same thing as it is implemented in programming code. Therefore, comparing currently compiled code of file and its bytecode (created much time earlier) will be good enhancement in your project.

Best regards,
Mehti

**Comment by myint** [comment]
Just to clarify, do you mean something like the following?

```
$ pydiff foo.py foo.pyc
```

**Comment by b5y** [comment]
Yes. It is exactly what I meant. Sorry for not giving example before. This idea came to my mind after reviewing [this](https://github.com/MyNameIsMeerkat/pyREtic) [tools](https://github.com/haypo/bytecode).
